src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -32,33 +32,42 @@ import vmAffectedBulkDetails from 'images/security_screenshot-affected-bulk-deta
 
 import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'
 
-New Relic Vulnerability Management may detect a good amount of vulnerabilities across your system. Most of these vulnerabilities may need remediation, but some might be not be applicable to your environment, or your internal policies might categorize them as low risk or limited exposure due to how your systems leverage the potentially vulnerable component.
+New Relic Vulnerability Management can detect a number of vulnerabilities across your system. While most of these vulnerabilities may need remediation, some might be not be applicable to your environment. For example, your internal policies might categorize them as low risk or limited exposure due to how your systems leverage the potentially vulnerable component.
 
-Understanding that you may want to tailor your visibility by entity or criticality, New Relic Vulnerability Management allows you to manage your vulnerabilities by:
+For vulnerabilities you're not interested in, you can set the **Status** of **Ignored** so they no longer appear in the UI. Since each vulnerability is tied to one or more [entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#what-is-entity), you can do the following:
```

Comment:
```suggestion
For vulnerabilities you're not interested in, you can set the **Status** to **Ignored** so they no longer appear in the UI. Since each vulnerability is tied to one or more [entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#what-is-entity), you can do the following:
```
```suggestion
For vulnerabilities you're not interested in, you can set the **Status** of **Ignored** so they no longer appear in the UI. Since each vulnerability is tied to one or more [entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#what-is-entity), you can do the following:
```